src/main/java/frc/robot/RobotContainer.java

```diff
@@ -150,6 +154,8 @@ private void configureBindings() {
 
     m_driverController.x().whileTrue(prepCoralIntake()).onFalse(coralHandoff());
 
+    IS_TELEOP.and(IS_REEFMODE).onTrue(prepScoreCoral(0, 220));
```

Comment:
Do we always start the prepScoreCoral whenever we are in reef mode?
Seems like we won't have good control of when it starts.

We should also pass in the official constants for the target height - and also want some ability to score on l2 / l3